### ISSUE TITLE
Quagga failed to create multipath for IPv6 routes

### ISSUE THREAD

**Issue by stcheng** [sonic-net/sonic-quagga#11]
```
str-s6000-on-2# show ipv6 bgp 20c0:a82e:0:71::
BGP routing table entry for 20c0:a82e:0:71::/64
Paths: (8 available, best #8, table Default-IP-Routing-Table)
  Advertised to non peer-group peers:
  fc00::2 fc00::a fc00::12 fc00::22 fc00::2a fc00::32 fc00::3a fc00::42 fc00::46 fc00::4a fc00::4e fc00::52 fc00::56 f
c00::5a fc00::5e fc00::62 fc00::66 fc00::6a fc00::6e fc00::72 fc00::76 fc00::7a fc00::7e
  65200 62047 65508
    fc00::a from fc00::a (100.1.0.3)
    (fe80::5054:ff:fedb:ccb4)
      Origin incomplete, localpref 100, valid, external
      Last update: Wed May  3 22:05:00 2017

  65200 62047 65508
    fc00::32 from fc00::32 (100.1.0.13)
      Origin incomplete, localpref 100, valid, external, multipath
      Last update: Wed May  3 22:04:29 2017

  65200 62047 65508
    fc00::3a from fc00::3a (100.1.0.15)
    (fe80::5054:ff:fee1:e6bc)
      Origin incomplete, localpref 100, valid, external
      Last update: Wed May  3 22:04:29 2017

  65200 62047 65508
    fc00::12 from fc00::12 (100.1.0.5)
    (fe80::5054:ff:fe03:f9a4)
      Origin incomplete, localpref 100, valid, external
      Last update: Wed May  3 22:04:28 2017

  65200 62047 65508
    fc00::2 from fc00::2 (100.1.0.1)
    (fe80::5054:ff:fe67:ca1a)
      Origin incomplete, localpref 100, valid, external
      Last update: Wed May  3 22:04:28 2017

  65200 62047 65508
    fc00::22 from fc00::22 (100.1.0.9)
    (fe80::5054:ff:fecf:9dd1)
      Origin incomplete, localpref 100, valid, external
      Last update: Wed May  3 22:04:28 2017

  65200 62047 65508
    fc00::2a from fc00::2a (100.1.0.11)
    (fe80::5054:ff:fe4c:f400)
      Origin incomplete, localpref 100, valid, external
      Last update: Wed May  3 22:04:28 2017

  65200 62047 65508
    fc00::1a from fc00::1a (100.1.0.7)
      Origin incomplete, localpref 100, valid, external, multipath, best
      Last update: Wed May  3 22:04:27 2017
```

Only two next hops become the multipath and are syncd to the routing table and then kernel.

**Comment by lguohan** [comment]
is this only applicable to ipv6 or it also applies to ipv4?

**Comment by stcheng** [comment]
It is only seen on IPv6.

**Comment by pavel-shirshov** [comment]
can you please share your bgp configuration?

**Comment by lguohan** [comment]
resolved in #20
